docs/source/documentation.rst

```diff
@@ -2,8 +2,8 @@ Documentation
 =============
```

Comment:
So, for some reason the documentation on the ReadTheDocs build is not working, but turns out that this is an issue for all branches except the latest master documentation. The documentation matches the required format and builds perfectly fine locally, so I expect it will also do so on the ReadTheDocs once merged.